src/main/resources/migrations/00060_create_table_govukpay_events.sql

```diff
@@ -0,0 +1,20 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_table-govukpay_events
+CREATE TABLE govukpay_events (
+    id BIGSERIAL PRIMARY KEY,
+    mandate_id BIGINT,
+    payment_id BIGINT,
+    event_date TIMESTAMP WITH TIME ZONE DEFAULT (now() AT TIME ZONE 'utc') NOT NULL,
+    resource_type VARCHAR(255),
+    event_type VARCHAR(255)
```

Comment:
This is the what happened bit, right? GoCardless call this an action. Maybe because it actually _is_ the event (conceptually), we could call it a name or something. But what’s here will do.

Reply:
i thought i'd go with what ledger calls things for some level of consistency

Reply:
Ah, I didn’t know about ledger using the name. That’s cool.